src/NetscriptFunctions/Bladeburner.ts

```diff
@@ -326,7 +326,7 @@ export function NetscriptBladeburner(
       checkBladeburnerCity("getCityEstimatedCommunities", cityName);
       const bladeburner = player.bladeburner;
       if (bladeburner === null) throw new Error("Should not be called without Bladeburner");
-      return bladeburner.cities[cityName].commsEst;
+      return bladeburner.cities[cityName].comms;
```

Comment:
The function should be renamed to `getCityCommunities` because it's no longer an estimate, all the strings should be changed as well.

The documentation for that function needs to be updated as well.